sdk/clientcore/core/src/main/java/io/clientcore/core/traits/HttpTrait.java

```diff
@@ -52,23 +52,6 @@ public interface HttpTrait<T extends HttpTrait<T>> {
      */
     T httpClient(HttpClient client);
 
-    /**
-     * Sets the {@link HttpPipeline} to use for the service client.
-     *
-     * <p><strong>Note:</strong> It is important to understand the precedence order of the {@link HttpTrait} APIs. In
-     * particular, if a {@link HttpPipeline} is specified, this takes precedence over all other APIs in the trait, and
-     * they will be ignored. If no {@link HttpPipeline} is specified, an HTTP pipeline will be constructed internally
-     * based on the settings provided to this trait. Additionally, there may be other APIs in types that implement this
-     * trait that are also ignored if an {@link HttpPipeline} is specified, so please be sure to refer to the
-     * documentation of types that implement this trait to understand the full set of implications.</p>
-     *
-     * @param pipeline {@link HttpPipeline} to use for sending service requests and receiving responses.
-     *
-     * @return Returns the same concrete type with the appropriate properties updated, to allow for fluent chaining of
-     * operations.
-     */
-    T httpPipeline(HttpPipeline pipeline);
```

Comment:
You'll need to set Core V2 and Identity V2 to use unreleased versions due to this API being removed.